cmd/syncEnumerator.go

```diff
@@ -211,6 +213,14 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 	}
 }
 
+func IsLocationCaseInsensitive(fromTo common.FromTo) bool {
```

Comment:
This function should be a method of common.Location or have common.Location as argument instead of FromTo. We're interested in destination alone, and the name is a misnomer.